### ISSUE TITLE
detail config of ablation studies

### ISSUE THREAD

**Issue by JasonWong4332** [ymy-k/DPText-DETR#23]
Hello,  can you share the detailed configuration of the ablation experimentation? 
I have conducted several experiments by using fewer training iterations and fewer training data, but the model performance can't reach the digit reported in the paper.

**Comment by ymy-k** [comment]
What's the configuration you used? And what about the performance diffrence between baseline w/ pos.label and baseline w/pos.label+EPQM+EFSA (or baseline w/pos.label+EPQM )?

**Comment by ymy-k** [comment]
What's the configuration you used? And how about the performance diffrence between baseline w/ pos.label and baseline w/pos.label+EPQM+EFSA (or baseline w/pos.label+EPQM )?

**Comment by JasonWong4332** [comment]
Thank you for your reply!
I train the model with pos.label, EPQM and EFSA with 12k iter using original total-text(td-ratio 100%) but only get 81.6 f-score on the original total-text test set.

the detailed configuration is as below:

_BASE_: "../Base.yaml"

DATASETS:
  TRAIN: ("totaltext_poly_train_pos",)  
  TEST: ("totaltext_poly_test",)  

MODEL:
  WEIGHTS: "detectron2://ImageNetPretrained/torchvision/R-50.pkl" 

SOLVER:
  IMS_PER_BATCH: 8
  BASE_LR: 1e-4
  LR_BACKBONE: 1e-5
  WARMUP_ITERS: 0
  STEPS: (9600,)
  MAX_ITER: 12000
  CHECKPOINT_PERIOD: 12000

TEST:
  EVAL_PERIOD: 3000

OUTPUT_DIR: "/data/jesonwong47/DPText-DETR/output/r_50_poly_25/totaltext"

**Comment by ymy-k** [comment]
Try to set INFERENCE_TH_THRESH from 0.4 to 0.3, referring to issue #18

**Comment by ymy-k** [comment]
How about the performance now? I just viewed the config. A mistake: the thresh was set to 0.35 for the models in table 4. There is also a difference about the STEPS. It was set to (10000,)

**Comment by JasonWong4332** [comment]
Thank you for the reminder, I will leave the result after the experiment.

**Comment by JasonWong4332** [comment]
the f-score is 83.67 after modifying the step and threshold  

> How about the performance now? I just viewed the config. A mistake: the thresh was set to 0.35 for the models in table 4. There is also a difference about the STEPS. It was set to (10000,)

**Comment by ymy-k** [comment]
OK，thanks, it's almost the same result.

**Comment by zhs1316** [comment]
> OK，thanks, it's almost the same result.

作者您好，我在更少迭代次数上进行了实验，但是hmean值只能达到81.08这是我的设置，请问还有别的地方需要修改吗

MODEL:
  META_ARCHITECTURE: "TransformerPureDetector"
  MASK_ON: False
  PIXEL_MEAN: [123.675, 116.280, 103.530]
  PIXEL_STD: [58.395, 57.120, 57.375]
  BACKBONE:
    NAME: "build_resnet_backbone"
  RESNETS:
    DEPTH: 50
    STRIDE_IN_1X1: False
    OUT_FEATURES: ["res3", "res4", "res5"]
  TRANSFORMER:
    ENABLED: True
    NUM_FEATURE_LEVELS: 4
    ENC_LAYERS: 6
    DEC_LAYERS: 6
    DIM_FEEDFORWARD: 1024
    HIDDEN_DIM: 256
    DROPOUT: 0.1
    NHEADS: 8
    NUM_QUERIES: 100
    ENC_N_POINTS: 4
    DEC_N_POINTS: 4
    USE_POLYGON: True
    NUM_CTRL_POINTS: 16
    EPQM: True
    EFSA: True
    INFERENCE_TH_TEST: 0.3

SOLVER:
  WEIGHT_DECAY: 1e-4
  OPTIMIZER: "ADAMW"
  LR_BACKBONE_NAMES: ['backbone.0']
  LR_LINEAR_PROJ_NAMES: ['reference_points', 'sampling_offsets']
  LR_LINEAR_PROJ_MULT: 0.1
  CLIP_GRADIENTS:
    ENABLED: True
    CLIP_TYPE: "full_model"
    CLIP_VALUE: 0.1
    NORM_TYPE: 2.0

INPUT:
  MIN_SIZE_TRAIN: (480, 512, 544, 576, 608, 640, 672, 704, 736, 768, 800, 832,)
  MAX_SIZE_TRAIN: 1600
  MIN_SIZE_TEST: 1000
  MAX_SIZE_TEST: 1800
  CROP:
    ENABLED: True
    CROP_INSTANCE: False
    SIZE: [0.1, 0.1]
  FORMAT: "RGB"

TEST:
  DET_ONLY: True  # evaluate only detection metrics

_BASE_: "./Base.yaml"

DATASETS:
  TRAIN: ("totaltext_poly_train_pos",)
  TEST: ("totaltext_poly_test",)  # or "inversetext_test", "totaltext_poly_test_rotate"

MODEL:
  WEIGHTS: "demo/weights/R-50.pkl"  # "detectron2://ImageNetPretrained/torchvision/R-50.pkl"  # or the provided pre-trained model

SOLVER:
  IMS_PER_BATCH: 8 # batch_size
  BASE_LR: 1e-4
  LR_BACKBONE: 1e-5
  WARMUP_ITERS: 0
  STEPS: (10000,)
  MAX_ITER: 12000
  CHECKPOINT_PERIOD: 12000

TEST:
  EVAL_PERIOD: 1000
